test/odpi_eunit.erl

```diff
@@ -1018,7 +1018,7 @@ ref_cursor([Context, Conn]) ->
         <<"">>
     ),
     ?assertEqual(0, dpi:stmt_execute(CreateStmt, [])),
-    ?assertEqual(ok, dpi:stmt_release(CreateStmt)),
+    ?assertEqual(ok, dpi:stmt_close(CreateStmt, <<>>), <<>>),
```

Comment:
I think this is wrong, shouldn't it be
    ?assertEqual(ok, dpi:stmt_close(CreateStmt, <<>>)),
without that second `, <<>>`?

Reply:
Good catch. How is it even compiling!

Reply:
http://erlang.org/doc/man/assert.hrl.html
Third / last param of assert macros are comment

Removing